tests/user_operation_erc4337.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import asdict, dataclass
+
+from eth_typing import HexStr
+from eth_utils import to_checksum_address
+
+from tests.transaction_eip_7702 import TupleEIP7702
+from tests.types import RPCRequest, CommandLineArgs
+
+
+@dataclass
+class UserOperation:
+    # pylint: disable=too-many-instance-attributes, invalid-name
+    sender: HexStr
+    nonce: HexStr = hex(0)
+    factory: HexStr = None
+    factoryData: HexStr = None
+    callData: HexStr = "0x"
+    callGasLimit: HexStr = hex(3 * 10**5)
+    verificationGasLimit: HexStr = hex(10**6)
+    preVerificationGas: HexStr = hex(3 * 10**5)
+    maxFeePerGas: HexStr = hex(4 * 10**9)
+    maxPriorityFeePerGas: HexStr = hex(3 * 10**9)
+    signature: HexStr = "0x"
+    paymaster: HexStr = None
+    paymasterData: HexStr = None
+    paymasterVerificationGasLimit: HexStr = None
+    paymasterPostOpGasLimit: HexStr = None
+    authorizationList: list[TupleEIP7702] = None
```

Comment:
just curious - for a single user operation, is that possible to attach multiple authorization tuples? and if so, what would be the use case?

https://hackmd.io/WJtBdwvnT5yLq90F7UilIQ is pointing a single authorization tuples UO structure which I personally think it makes lot of sense.

Reply:
Hey @andysim3d we're currently working on the validation rules, including whether we should allow more than one tuple. There are some use cases for that, but also some issues. We'll update soon